src/parse.c

```diff
@@ -1615,12 +1615,14 @@ handle_vxlan_guint(NetplanParser* npp, yaml_node_t* node, const void* data, GErr
     return handle_generic_guint(npp, node, npp->current.vxlan, data, error);
 }
 
-static gboolean
+// LCOV_EXCL_START
+static gboolean __attribute__ ((unused))
```

Comment:
I didn't want to remove this function, but I feel like I should...

Reply:
Yes, please remove